OSDOCS-16941: DITA abstracts — OLM catalog/config topics (#115366)
openshift/openshift-docs: modules/olm-software-catalog-architecture.adoc

[id="olm-software-catalog-arch_{context}"]
= Software catalog architecture

[role="_abstract"]

> The software catalog UI in {product-title} is driven by the Marketplace Operator, which manages default catalog sources through the cluster `OperatorHub` custom resource.

Reviewer: ```suggestion
The software catalog UI component is driven by the Marketplace Operator by default on {product-title} in the `openshift-marketplace` namespace.
```
